qualtran/bloqs/arithmetic/addition.py

```diff
@@ -280,7 +281,9 @@ def signature(self):
             [
                 Register('a', QUInt(self.bitsize)),
                 Register('b', QUInt(self.bitsize)),
-                Register('c', QUInt(self.bitsize + 1), side=side),
+                Register(
+                    'c', QUInt(self.bitsize + int(self.include_most_significant_bit)), side=side
```

Comment:
update Registers: doc as well

Reply:
consider doing ` + (1 if self.include_most_significant_bit else 0)` for explicitness

Reply:
done